design-system-docs/src/_foundations/typography.md

```diff
@@ -85,6 +85,25 @@ If you need to apply these styles to heading elements that you cannot add a clas
 </main>
 ```
 
+If you need to apply these styles to heading elements that you cannot add a class to but want more tightly scoped styles than the default `.cads-prose` class you can add a `.cads-prose-scoped` class instead. This is useful when you are embedding lots of rich components within a prose block and you want to more narrowly scope the effects of the cascade.
```

Comment:
I think we should be specific here and say it will be scoped using `>`, ie to the direct children of the element it's applied to.  It's implied in the example below but would be nice to be super clear.